src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  background-color: black;
+  margin: 0;
+}
+
+.box {
+  width: 350px;
+  background-color: #308dc0;
+  border: 1px solid #e75551;
+  margin: 20px;
+  margin-bottom: 40px;
+  box-sizing: border-box;
+  padding: 10px;
+}
+
+.orange_box {
+  background-color: #ffa500;
+  border: 1px solid #e75551;
+  padding: 10px;
```

Comment:
this pair of rules appears in three different classes
If we have 5 blocks and these blocks have common characteristics, such as height, width, color, fonts, etc., then we should move these characteristics in a separate class and use it for style these blocks. DRY - Don't Repeat Yourself.